ChatDemo/ViewController.swift

```diff
@@ -8,11 +8,22 @@
 
 import UIKit
 // Add Stream Chat SDK.
+import StreamChatClient
 import StreamChatCore
 import StreamChat
 
 /// Use ChannelsViewController as the parent view controller.
 class ViewController: ChannelsViewController {
+    required init?(coder: NSCoder) {
+        Client.configureShared(.init(apiKey: "m7cdet3kpfdu", logOptions: .info))
+        
+        let userExtraData = UserExtraData(name: "Square art")
+        Client.shared.set(user: User(id: "square-art-0", extraData: userExtraData),
+                          token: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoic3F1YXJlLWFydC0wIn0.HDTQ8dDHSndescdgmi-T6Ta0yORvVD6sYLXB08CqZaM")
+        
+        super.init(coder: coder)
+    }
+    
```

Comment:
Why did you decide for this change?

Reply:
If it's in AppDelegate, it runs after the ChannelsViewController is initiated, which calls Client.shared stuff and crashes because configure wasn't yet called. You can try it if you want. Let me know if there's a better way to handle it. Ofc, I could instantiate the view manually in AppDelegate, but that's not how most people will do it.

Reply:
Cool, I was just curious. Thank you for explaining 👍